src/dotnet/commands/dotnet-new/CSharp_nunittest/project.json.template

```diff
@@ -0,0 +1,21 @@
+{
+    "version": "1.0.0-*",
+
+    "dependencies": {
+        "dotnet-test-nunit": "3.4.0-beta-1"
+    },
+
+    "testRunner": "nunit",
+
+    "frameworks": {
+        "netcoreapp1.0": {
+            "imports": "portable-net45+win8",
```

Comment:
Is this imports necessary?  What under "dotnet-test-unit" isn't on "netstandard" yet?